Antti Kokko (1):
      Add changes file for Qt 5.15.2

Jani Heikkinen (2):
      Bump version
      Bump version

Paul Wicking (1):
      Doc: Remove mention of Mozilla Persona

Tarja Sundqvist (9):
      Update commercial license headers
      Revert "Update commercial license headers"
      Bump version
      Bump version
      Bump version
      Merge remote-tracking branch 'origin/tqtc/lts-5.15.4' into tqtc/lts-5.15-opensource
      Merge remote-tracking branch 'origin/tqtc/lts-5.15.5' into tqtc/lts-5.15-opensource
      Merge remote-tracking branch 'origin/tqtc/lts-5.15.6' into tqtc/lts-5.15-opensource
      Merge remote-tracking branch 'origin/tqtc/lts-5.15.7' into tqtc/lts-5.15-opensource

diff --git a/src/oauth/doc/src/qtnetworkauth.qdoc b/src/oauth/doc/src/qtnetworkauth.qdoc
index c2ab112..3810d79 100644
--- a/src/oauth/doc/src/qtnetworkauth.qdoc
+++ b/src/oauth/doc/src/qtnetworkauth.qdoc
@@ -66,7 +66,6 @@
       \li \l {https://tools.ietf.org/html/rfc6749} {OAuth 2}
       \li \l {http://openid.net} {OpenID}
       \li \l {http://openid.net/connect/} {OpenID Connect}
-      \li \l {https://developer.mozilla.org/Persona} {Mozilla Persona}
     \endlist
 
     These systems allow the application developers to create applications which
